code/game/machinery/cloning.dm

```diff
@@ -109,7 +111,27 @@ GLOBAL_VAR_INIT(clones, 0)
 	. = ..()
 	. += "The write-protect tab is set to [read_only ? "protected" : "unprotected"]."
 
+// Biomass
 
+/obj/machinery/clonepod/attackby(obj/item/W, mob/user, params)
+	var/old_biomass = biomass // saves biomas variable
+	if(istype(W, /obj/item))
+		if(istype(W, /obj/item/stack/sheet/animalhide/human)) // lets play flesh or meat
+			biomass += 50
+		if(istype(W, /obj/item/reagent_containers/food/snacks/meat/slab)) // Are we inserting meat?
+			if(istype(W, /obj/item/reagent_containers/food/snacks/meat/slab/human)) // human meat is the easiest to turn into new human materials
+				biomass += 50
+			else if(istype(W, /obj/item/reagent_containers/food/snacks/meat/slab/synthmeat))
+				biomass += 34 // synthmeat can be many different things, thus it should be decently high. This ensures 3 of them gives you a full clone, without fucking with decimals.
+			else if(istype(W, /obj/item/reagent_containers/food/snacks/meat/slab/monkey))
+				biomass += 25 // Monkey meat is close to human, but not actually human.
+			// if(istype W, /obj/item/reagent_containers/) //  this space will eventually be for my biomass cartidge, which aren't done.
+			else
+				biomass = biomass + 20 // Not actually human meat? Means that you need more of it.
```

Comment:
```suggestion
				biomass += 20 // Not actually human meat? Means that you need more of it.
```

Reply:
This else is exploitable, since any /obj/item (aka 75% of the game) will turn into 20 biomass